src/client/datascience/notebook/integration.ts

```diff
@@ -75,8 +75,8 @@ export class NotebookIntegration implements IExtensionSingleActivationService {
                                 runnable: true,
                                 executionOrder: false,
                                 custom: false,
-                                runState: false,
-                                statusMessage: false
+                                runState: true,
+                                statusMessage: true
```

Comment:
These should never cause the document to be dirty as they are not persisted in the ipynb file.
These are used only to update the UI (at least in our implementation of notebooks)